src/components/date-picker-month/date-picker-month.tsx

```diff
@@ -73,24 +73,24 @@ export class DatePickerMonth {
   /**
    * Event emitted when user selects the date.
    */
-  @Event() calciteDatePickerSelect: EventEmitter;
+  @Event() calciteDatePickerSelect: EventEmitter<void>;
 
   /**
    * Event emitted when user hovers the date.
    *
    * @internal
    */
-  @Event() calciteInternalDatePickerHover: EventEmitter;
+  @Event() calciteInternalDatePickerHover: EventEmitter<void>;
 
   /**
    * Active date for the user keyboard access.
    */
-  @Event() calciteDatePickerActiveDateChange: EventEmitter;
+  @Event() calciteDatePickerActiveDateChange: EventEmitter<void>;
```

Comment:
emit type should be `Date` right?

Reply:
considering `dateFromRange( )` return type could be `Date`

Reply:
Applies to following events as well.
`calciteInternalDatePickerMouseOut`
`calciteInternalDatePickerHover`
`calciteDatePickerSelect`

Reply:
Looks like it. Unless we want to encourage the user getting these values from the target which might be preferred.

Reply:
Good catch. The build didn't complain and I hadn't checked the test results. Looks like I also left out some commits. Will fix. 

Sidebar: it looks like all of the events from date-picker supporting components are really internal since they're not exposed and they are not mapped on the date-picker itself (meaning not in the date-picker type snor doc).

Reply:
make sense why they didn't show up in the build error.